certify-core/src/main/java/io/mosip/certify/core/util/LinkCodeQueue.java

```diff
@@ -0,0 +1,28 @@
+/*
```

Comment:
This class is required? Let's discuss if there is any dependency.

Reply:
removed this class